pulsar/internal/metrics.go

```diff
@@ -21,9 +21,8 @@ import (
 	"github.com/prometheus/client_golang/prometheus"
 )
 
-var topicLabelNames = []string{"pulsar_tenant", "pulsar_namespace", "topic"}
-
 type Metrics struct {
+	metricsLevel      int
```

Comment:
Would it make sense to define a 2nd enum for internal use instead of ints?

Reply:
It might, but that’s definitely a hack around some layout problems. But there probably aren’t any half-measures there, save an awkward new package, so I guess that’s a ticket for another day.

Reply:
One option could be to make the consts be strings instead of integers, and then have `internal` do a switch on these. Then at least the hard-coded values would have more meaning than inscrutable integer values.